app/assets/javascripts/partners.js

```diff
@@ -0,0 +1,12 @@
+$(document).on('turbolinks:load', function() {
+    $('form').on('change', '#partner_agency_type', function(event) {
+        console.log("I changed successfully");
+        console.log($(this).val() == "Other");
```

Comment:
we can remove these `console.log` statements now

Reply:
Ha! Thanks @chaserx !!